apps/judicial-system/digital-mailbox-api/src/app/modules/cases/models/subpoena.response.ts

```diff
@@ -121,8 +122,8 @@ export class SubpoenaResponse {
         subtitle: courtNameAndAddress,
         hasBeenServed: hasBeenServed,
         hasChosenDefender: Boolean(
-          defendantInfo?.defenderChoice &&
-            defendantInfo.defenderChoice !== DefenderChoice.DELAY,
+          defendantInfo?.requestedDefenderChoice &&
+            defendantInfo.requestedDefenderChoice !== DefenderChoice.DELAY,
```

Comment:
It shouldn't really be needed though since if the defendantInfo is undefined the first part would be false and it should stop doing checks at that point